src/main/java/gg/solarmc/kitpvp/stat/PlaceholderAPIHook.java

```diff
@@ -0,0 +1,49 @@
+package gg.solarmc.kitpvp.stat;
+
+import gg.solarmc.kitpvp.KitpvpPlugin;
+import gg.solarmc.loader.kitpvp.KitPvpKey;
+import gg.solarmc.loader.kitpvp.OnlineKitPvp;
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.entity.Player;
+
+public class PlaceholderAPIHook extends PlaceholderExpansion {
```

Comment:
For now, don't add placeholder support with PAPI. Trust me when I say the placeholder API we use will be much better and cleaner, so we will not have to link our code with ugly calls to PlaceholderAPI.setPlaceholders. Ideally our plugin code will stay separate from the mess of PAPI, but our placeholder framework can be connected to PAPI via a bridge implementation.

Reply:
i'll keep the class until then so that ariel can test